src/Lykke.Service.Decred.Api/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Lykke.Service.Decred.Api
+{
+    internal sealed class Program
```

Comment:
Use Lykke.Sdk, as it done in sign service

Reply:
I've explained my case on Telegram, this introduces an issue that needs to be addressed by Lykke.  I will not work on this.